run_tests.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+import feedparser
+import sys
+import os
+
+feed = feedparser.parse('./feed.rss')
+
+absolute_address = "https://contravariance.rocks/"
+
+def validate_path(path):
+    replaced = path.replace(absolute_address, "")
+    # If file doesn't exist or is empty, return false
+    return os.path.getsize(replaced) != 0
+
+def fail(message):
+    print message
+    sys.exit(1)
+
+for entry in feed.entries:
+    print "Validating", entry.title
+    if not validate_path(entry["image"]["href"]):
+        fail("Could not find image %s" % (entry["image"]))
+    for enclosure in entry.enclosures:
+        if int(enclosure["length"] == 0):
+            fail("Invalid length for item '%s'" % (entry.title,))
+        if not validate_path(enclosure["href"]):
+            fail("Could not find mp3 '%s' for '%s'" % (enclosure["href"], entry.title,))
+    (_, minutes, _) = entry["itunes_duration"].split(":")
+    if int(minutes) == 0:
```

Comment:
What are you trying to validate here, exactly? This would fail if an episode is 1 hour and 0 minutes, I think, which to me is unexpected.

Reply:
Well, just checking if the output is kinda the right format (in a cheap manner). I'll make it so that 1 hour 0 minutes also works :)